frontend/src/app/libs/common-components/components/button/button.component.html

```diff
@@ -32,6 +32,19 @@
 </ng-container>
 
 <ng-template #buttonText>
-	<mat-icon *ngIf="icon" class="button__icon">{{icon}}</mat-icon>
-	<ng-content></ng-content>
+	<div class="button__content">
+		<mat-icon
+			*ngIf="icon"
+			class="button__icon"
+			[class.button__icon--extra-small]="isIconExtraSmall"
```

Comment:
i think you can add class based on iconSize input, like class="button__icon--{{iconSize}}"